public/thread.php

```diff
@@ -1,7 +1,7 @@
 <?php require('header.php'); ?>
 
 
-<form>
+<form action="post_reply.php" method="post">
```

Comment:
So you kind of made this file into a "post thread" **and** a "post reply" page. Which doesn't really make sense. But either way, this action belongs in the form for posting a reply, which is on line 19.